src/engine/value.rs

```diff
@@ -261,6 +266,46 @@ where
     }
 }
 
+impl TryFrom<Value> for HashMap<String, Value> {
+    type Error = Error;
+
+    fn try_from(value: Value) -> Result<HashMap<String, Value>, Error> {
+        match value {
+            Value::Map(hm) => Ok(hm),
+            _ => Err(Error::TypeConversionFailed {
+                src: format!("{:#?}", value),
+                dst: "HashMap<String, Value>".to_string(),
+            }),
+        }
+    }
+}
+
+/*
```

Comment:
In general, I try to avoid commented out code going into master branches.  Is there a reason to keep this, or should it be deleted?

Reply:
That was an oversight. Fixed.